Documentation/markdown/CMakeOptions.md

```diff
@@ -4,17 +4,21 @@ This lists all CMake Options that can be set for the TUM Open Infra Platform pro
 ## Overview of the CMake Options
 
 A CMake Option is visible in the CMake GUI as a box that can be ticked (or not). 
-If you have made changes and want to return to the default values, you will have to delete the CMake cache and configure again. Refer to the CMake "Option" command [here](https://cmake.org/cmake/help/latest/command/option.html).
+If you have made changes and want to return to the default values, you will have to delete the CMake cache and configure again. 
+Refer to the CMake "Option" command [here](https://cmake.org/cmake/help/latest/command/option.html).
 
-The options that can be set for the TUM Open Infra Platform are grouped as project-specific options and external options (libraries and thirdparty software). For each, you will find a description of what each option does, what its default value is and where to find the source code where the option is set if you require more information or wish to make changes there. 
+The options that can be set for the TUM Open Infra Platform are grouped as project-specific options and external options (libraries and thirdparty software). 
+For each, you will find a description of what each option does, what its default value is and where to find the source code where the option is set if you require more information or wish to make changes there. 
 Click on the name below to jump to the respective section. 
 
 - [Project options](#project-options)
 - [External options](#external-options)
 
 ## Project options
 
-The following are options specific to the build of the TUM Open Infra Platform project. The options are grouped thematically. Click on the name below to jump to the respective section. If you want to see information about external library and thirdparty software options instead, you can find them [here](#external-options). 
+The following are options specific to the build of the TUM Open Infra Platform project. 
+The options are grouped thematically. Click on the name below to jump to the respective section. 
+If you want to see information about external library and thirdparty software options instead, you can find them [here](#external-options). 
```

Comment:
please put this sentence after the list of sections.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/111/commits/1c933137bd94fddfb2a536635359b4a839f8c791